File: CS312-Principles_of_Programming/Assignment_5/Prog1.cpp
#include <iostream>
double q( int a, int b, int c ){
    double q;
    c = a + b + c;
    a = 2 * c;
    b = b * 4;
    q = (double)a / (double)b;
    std::cout << a <<" " << b << " "<< c <<" " << q << "\n";
    return q;
}

int main(){
    int x, y, z;
    double i;
    printf("Enter three integers: ");
    x = 3;
    y = 4;
    z = 5;
    i = q(x, y, z);
    std::cout << "result: " << x <<" " << y << " " << z <<" " << i;
}

File: CS312-Principles_of_Programming/Assignment_5/Prog1.pas
uses sysutils;
function q(var a: integer; b : integer; var c : integer):real;
  begin
    c:=a+b+c;
    a:=2*c;
    b:=b*4;
    q:=a/b;
  end;

var
  x, y, z : integer;
  i : real;
begin
  writeln('Enter three integers: ');
  x := 3;
  y := 4;
  z := 5;
  i := q(x, y, z);
  writeln('result: ', x, ' ', y, ' ', z, ' ', i);
end.
